pages/resources.js

```diff
@@ -1,15 +1,160 @@
+import Link from "next/link";
+import { useState } from "react";
 import styled from "styled-components";
 
 const Resources = () => {
   return (
     <div>
       <Headline>Hilfestellen und Beratungsangebote</Headline>
       <List>
-        <ListItem>Fachberatung1</ListItem>
-        <ListItem>Fachberatung2</ListItem>
-        <ListItem>Fachberatung3</ListItem>
-        <ListItem>Fachberatung4</ListItem>
-        <ListItem>Fachberatung5</ListItem>
+        <ListItem>
+          <h3>bff: Gewalt gegen Frauen e.V.</h3>
+          <Link href="https://www.frauen-gegen-gewalt.de" passHref>
+            bff: Gewalt gegen Frauen e.V.
+          </Link>
+        </ListItem>
```

Comment:
It‘s better to separate your UI logic from your data.

You could create an array of objects, each containing a title, link and description field.

Then you can map over the array and create a <ListItem> for every object with the corresponding data.